README.md

```diff
@@ -6,16 +6,59 @@ Call the Harness.io API to deploy an application.
 
 ## Usage
 
+```yaml
+   - name: Deploy
+     id: deploy
+     uses: fac/harness-deploy-action@v1
+     with:
+       webhookUrl: ${{ secrets.HARNESS_WEBHOOK_URL }}
+       application: XYZ12345678ABC
+       services: web,console,job
+       version: ${{ github.sha }}
+
+   - name: Deploy Link
+     run: |
+       echo harness_url: ${{ steps.deploy.outputs.harness_url }}
+```
+
 ## Inputs
 
+### webhookUrl
+
+The full webhook url for harness, including the authentication token. You **should** pass this as a secret.
```

Comment:
drop the should, 'Add the URL as an Actions secret on your repo, and pass it to the action'